src/components/pages/WelcomePage.tsx

```diff
@@ -763,19 +765,33 @@ class WelcomePage extends React.Component<Props, State> {
   private setAutocompleteOptions = async (path: string, input: string) => {
 
     // Handle place autocomplete
-    if (path === "/linkedevents/places/search") {
-      if (input.length < 3) {
-        return [];
-      }
-      const res = await fetch(`https://mantyharju-test.linkedevents.fi/v1/search/?type=place&input=${ input }`);
-      const data = await res.json();
-      return data.data.map((place: any) => {
-        return {
-          name: place.name && place.name.fi ? place.name.fi : place.id,
-          value: place.id
-        };
-      });
+    if (input.length < 1) {
```

Comment:
Could this just be `if (!input)` since empty array is falsey?

Reply:
Changed